griptape/drivers/embedding/azure_openai_embedding_driver.py

```diff
@@ -17,14 +18,22 @@ class AzureOpenAiEmbeddingDriver(OpenAiEmbeddingDriver):
         tokenizer: Optionally provide custom `OpenAiTokenizer`.
     """
 
-    model: str = field(kw_only=True)
     deployment_id: str = field(kw_only=True)
     api_base: str = field(kw_only=True)
     api_type: str = field(default="azure", kw_only=True)
     api_version: str = field(default="2023-05-15", kw_only=True)
     tokenizer: OpenAiTokenizer = field(
         default=Factory(lambda self: OpenAiTokenizer(model=self.model), takes_self=True), kw_only=True
     )
+    client: openai.AzureOpenAI = field(
+        init=False,
```

Comment:
Why `init=False`?

Reply:
Could you please also add it to the docstring?

Reply:
Accidental, removed and updated docstrings